python/tvm/relax/frontend/nn/modules.py

```diff
@@ -612,6 +621,154 @@ def append(self, new_element: Tensor) -> None:
         )
 
 
+class PagedKVCache(Object):
```

Comment:
We might want to think of temporarily leaving this KVCache object to mlc for fast development if its design is still in flux

Reply:
Yah agree with you!

Reply:
Do you have a good idea on where it should be placed under MLC? I do think it will be tweaked for a few times.

Reply:
How about `python/mlc_chat/model/utils/kv_cache.py`?

Reply:
Sounds good, just updated. The unit test for PagedKVCache will be moved to MLC accordingly.